SageJournals.js

```diff
@@ -0,0 +1,389 @@
+{
+	"translatorID": "908c1ca2-59b6-4ad8-b026-709b7b927bda",
```

Comment:
This translator will cause a conflict when merging, similar translators "SAGE Journals.js" with same guid + label exists